test_gardenpath fails on OpenIndiana

The `test_gardenpath` test fails on OpenIndiana because the string returned from `strerror()` is implementation specific.

Simple test case in C:
```
#include <stdio.h>
#include <string.h>
#include <errno.h>

int main(void)
{
        printf("%s\n", strerror(EPERM));
        return 0;
}
```
On linux it prints: `Operation not permitted`, but on OpenIndiana (and thus on all illumos based distros, and likely Solaris too) it prints: `Not owner`.

Here is the test failure:
```
________________________ Test__strerror.test_gardenpath ________________________

self = <tests.test_wasyncore.Test__strerror testMethod=test_gardenpath>

    def test_gardenpath(self):
>       self.assertEqual(self._callFUT(1), "Operation not permitted")

tests/test_wasyncore.py:1199:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/lib/python3.9/vendor-packages/teamcity/diff_tools.py:33: in _patched_equals
    old(self, first, second, msg)
E   AssertionError: 'Not owner' != 'Operation not permitted'
E   - Not owner
E   + Operation not permitted
```

--- a/tests/test_wasyncore.py
+++ b/tests/test_wasyncore.py
@@ -1196,7 +1196,10 @@ class Test__strerror(unittest.TestCase):
         return _strerror(err)
 
     def test_gardenpath(self):
-        self.assertEqual(self._callFUT(1), "Operation not permitted")
+        from errno import EINVAL
+        from os import strerror
+
+        self.assertEqual(self._callFUT(EINVAL), strerror(EINVAL))
 
     def test_unknown(self):
         self.assertEqual(self._callFUT("wut"), "Unknown error wut")

